specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2022-05-01-privatepreview/common-types.json

```diff
@@ -0,0 +1,20 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Common types"
+  },
+  "paths": {},
+  "definitions": {
+    "Password": {
```

Comment:
Move this definition to respective file to avoid error related to common-types.json